connectors/src/connectors/github/lib/github_api.ts

```diff
@@ -883,7 +883,8 @@ export async function processRepository({
         sizeBytes: size,
         documentId,
         parentInternalId,
-        parents: [documentId, ...parents.map((p) => p.internalId)],
+        /// we reverse the parents here since the convention is bottom to top
+        parents: [documentId, ...parents.map((p) => p.internalId).reverse()],
```

Comment:
would do that above, using unshift in the parents computation loop, so we never have reverse() and we always have parents in the right order (changing parentInternalId too)

Reply:
Hmm it kind of messes up the other locations where we assume that parents are left to right (bc extracted from the path to the file), in this specific file I see the bottom to top order as a contentNode-specific thing, happy to discuss ofc